custom_components/babybuddy/services.yaml

```diff
@@ -147,143 +93,184 @@ services_feedings_add:
+            - Wet
+            - Solid
+    color:
+      name: Diaper color
+      description: Daiper color
+      selector:
+        select:
+          options:
+            - Black
+            - Brown
+            - Green
+            - Yellow
+    amount:
+      name: Diape change amount
+      description: Diaper change amount
+      selector:
+        number:
+          min: 1
+          max: 10
+          mode: box
     notes:
-      name: Sleep notes
-      description: Sleep notes
+      name: Change notes
+      description: Change notes
       selector:
         text:
+          multiline: true
 
-services_temperature_add:
+add_temperature:
   name: Add Temperature
-  description: Adds a temperature
+  description: Adds a temperature entry
+  target:
+    entity:
+      integration: babybuddy
+      domain: sensor
+      device_class: babybuddy__child
   fields:
-    child:
-      name: Child
-      description: Child to add a temperature to
-      required: true
-      selector:
-        entity:
     temperature:
       name: Temperature
-      description: Temperature in number format
+      description: Temperature
       required: true
       selector:
-        text:
+        number:
+          min: 35.0
+          max: 150.0
+          step: 0.1
+          mode: box
     time:
       name: Temperature time
-      description: Temperature time in YYYY-MM-DD HH:MM format
-      required: true
+      description: Temperature time
       selector:
-        text:
+        time:
     notes:
       name: Temperature notes
       description: Temperature notes
       selector:
         text:
+          multiline: true
 
-services_tummy_times_add:
-  name: Add Tummy Time
-  description: Adds Tummy Time
-  fields:
-    child:
-      name: Child
-      description: Child to add Tummy Time to
-      required: true
-      selector:
-        entity:
-    start:
-      name: Tummy Time start time
-      description: Tummy Time start time in YYYY-MM-DD HH:MM format
-      required: true
-      selector:
-        text:
-    end:
-      name: Tummy Time end time
-      description: Tummy Time end time in YYYY-MM-DD HH:MM format
-      required: true
-      selector:
-        text:
-    milestone:
-      name: Tummy Time milestone
-      description: Tummy Time milestone
-      selector:
-        text:
-
-services_weight_add:
+add_weight:
   name: Add Weight
-  description: Adds a weight
+  description: Adds a weight entry
+  target:
+    entity:
+      integration: babybuddy
+      domain: sensor
+      device_class: babybuddy__child
   fields:
-    child:
-      name: Child
-      description: Child to add a weight to
-      required: true
-      selector:
-        entity:
     weight:
       name: Weight
-      description: Weight in number format
+      description: Weight value
       required: true
       selector:
-        text:
+        number:
+          min: 0.0
+          max: 100.0
+          step: 0.1
+          mode: box
```

Comment:
why max here?

Reply:
This is required when using `number` selector in services.yaml

Reply:
hmm. doesn't seem to stop anything, just change some css styling. I think it's fine for now.